addons/a11y/src/components/Report/RerunButton.js

```diff
@@ -0,0 +1,17 @@
+import glamorous from 'glamorous';
+
+const RerunButton = glamorous.button({
+  position: 'absolute',
+  bottom: 0,
+  right: 0,
+  border: 'none',
+  borderTop: 'solid 1px rgba(0, 0, 0, 0.2)',
+  borderLeft: 'solid 1px rgba(0, 0, 0, 0.2)',
+  background: 'rgba(255, 255, 255, 0.5)',
+  padding: '5px 10px',
+  borderRadius: '4px 0 0 0',
+  color: 'rgba(0, 0, 0, 0.5)',
+  outline: 'none',
```

Comment:
This disables the outline on the button, which causes problems with focus.

Reply:
Ah, made it consistent with other places, will remove the `outline` line

Reply:
Let's do that in other places as well =) I wonder if it's possible to have an eslint rule for that